test/11_array.jl

```diff
@@ -650,4 +650,24 @@ end
     @test unique(x) == collect(1:10)
 end
 
+using JSON3
```

Comment:
Better put this in 13_arraycommon.jl, and test both `CategoricalVector{String}` and `CategoricalVector{Union{String, Missing}}` at the same time.